services/workflows-service/src/customer/customer.service.ts

```diff
@@ -1,13 +1,36 @@
 import { Injectable } from '@nestjs/common';
 import { CustomerRepository } from '@/customer/customer.repository';
 import { Prisma } from '@prisma/client';
+import { ApiKeyService } from '@/customer/api-key/api-key.service';
+import { generateApiKey } from '@/customer/api-key/utils';
 
 @Injectable()
 export class CustomerService {
-  constructor(protected readonly repository: CustomerRepository) {}
+  constructor(
+    protected readonly repository: CustomerRepository,
+    protected readonly apiKeyService: ApiKeyService,
+  ) {}
 
   async create(args: Parameters<CustomerRepository['create']>[0]) {
-    return this.repository.create(args);
+    // @ts-expect-error - prisma json not updated
+    const authValue = args.data?.authenticationConfiguration?.authValue;
```

Comment:
Why do you still use the old schema here?

Reply:
will remove old schema in the future 
https://linear.app/ballerine/issue/BAL-1757